packages/nextjs/src/client/index.tsx

```diff
@@ -67,26 +101,29 @@ export function ClerkProvider({ children, ...rest }: NextClerkProviderProps): JS
     };
   }, []);
 
+  const nextProps = {
+    frontendApi: frontendApi || process.env.NEXT_PUBLIC_CLERK_FRONTEND_API || '',
```

Comment:
🔧 We can remove the frontendApi since it's consider deprecated

Reply:
@SokratisVidros  I think this will be a breaking change for people still using `frontendApi` and having a caret  (`^`)  in their `package.json` for `@clerk/nextjs`. After releasing this, their next build will break.

Reply:
Can we mark it as deprecated then?